src/documentation/advanced_tutorials/100_deployments.page

```diff
@@ -5,33 +5,33 @@ sort_info: 100
 
 Conceptually, the execution of Orocos tasks is split into two parts. First, the
 task encapsulates the so-called "computation" part of the component. It is
-declared and developed, but _how_ it should be executed is left out. Second, it
-is __deployed__: The task is associated with a _trigger mechanism_ that defines
+declared and developped, but _how_ it should be executed is left out. Second, it
+is __deployed__: the task is associated with a _trigger mechanism_ that defines
 when the task's computation should be executed.
 
 Until now, you have used deployments implicitly as, for each task, oroGen
-defines a default deployment for convenience reasons. The default deployment
+defines a default deployment for convenience reasons. The default deployments
 execution schema is given in the task description by statements like
 [periodic(1.0)](../tutorials/110_basics_create_component.html) or
-[port_driven](../tutorials/130_basics_connect_components.html).
-Furthermore, the default deployments add a logging component by default and each
-task is run as a separate process.
+[port_driven "messages"](../tutorials/130_basics_connect_components.html).
+Furthermore the default deployments add a logging component by default and each
+task is ran in its own process.
 
 In order to allow a more flexible definition of the execution schema and putting multiple
 tasks into one process, explicit deployments can also be specified. They
-override the default deployment of the task. For further information, also have a look at the
-[documentation page](../orogen/deployment.html).
+override the default deployment of the task. For further information, also have a look on the
+documentation page [here](/documentation/orogen/deployment.html).
 
-Deployments are specified in an orogen project, the same way as tasks are.
+Deployments are specified in an orogen project, the same way than tasks are.
```

Comment:
"same way as" is the correct form http://forum.wordreference.com/showthread.php?t=400069